skiko/src/commonMain/kotlin/org/jetbrains/skiko/context/ContextHandler.kt

```diff
@@ -49,7 +41,14 @@ internal abstract class ContextHandler(val layer: SkiaLayer) {
                 "OS: ${hostOs.id} ${hostArch.id}\n"
     }
 
-    protected open fun destroyContext() {
-        context?.close()
+    // throws RenderException if initialization of graphic context was not successful
+    fun draw() {
+        if (!initContext()) {
+            throw RenderException("Cannot init graphic context")
+        }
+        initCanvas()
+        canvas?.clear(if (layer.fullscreen && hostOs != OS.MacOS) -1 else clearColor)
```

Comment:
Why not combined 

canvas?.apply {
  clear()
  drawContent()
}

Reply:
Done